__tests__/app.test.js

```diff
@@ -100,3 +99,41 @@ describe("/api/users", () => {
       });
   });
 });
+describe("PATCH", () => {
+  describe("/api/reviews/:reviews_id", () => {
+    test("200: Update 'votes' property in review table with the number provided", () => {
+      const voteChange = { inc_votes: 2 };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(voteChange)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.review.votes).toBe(3);
+        });
+    });
+  });
+  describe("/api/reviews/:reviews_id", () => {
+    test("404: returns an error message when client tries to vote for review_id that doesn't exist", () => {
+      const voteChange = { inc_votes: 2 };
+      return request(app)
+        .patch("/api/reviews/99999")
+        .send(voteChange)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404: Not found");
+        });
+    });
+  });
+  describe("/api/reviews/:reviews_id", () => {
+    test("404: returns an error message when client provideds invalid input", () => {
+      const voteChange = { inc_votes: 2 };
+      return request(app)
```

Comment:
sorry Ross I should have been clearer! by my previous error testing suggestion for invalid input. I meant that if we try to send a malformed body, so something like { inc_votes: "a billion"} across or {"wooooooo": 1}, the patch request should be rejected. This is to prevent naughty hackers from doing SQL injection on our lovely databases